test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -202,7 +203,8 @@ def check_loadTestsFromNames(names, module=None):
                                             "ptl", "tests")
                 user_test_dir = os.environ.get("PTL_TESTS_DIR", None)
                 if user_test_dir is not None:
-                    tests_dir += [user_test_dir]
+                    if os.path.isdir(user_test_dir):
```

Comment:
you can merge this if with above if: `if user_test_dir and os.path.isdir(user_test_dir):`